content/blog/2022-01-14-users-guide-to-contribution-ross/index.md

```diff
@@ -0,0 +1,45 @@
+
+Contributor motivations vary. Some work on Galaxy for a living. Others want to pay it forward or perhaps scratch an open source itch.
+
+Whatever motivates individuals, in addition to the satisfaction of contributing to an important open science resource, they gain valuable professional experience and skills, collaborating with talented and friendly people, working on problems they care about.
+
+**If this sounds interesting, please read on to learn more about contributing to Galaxy**
+
+---
+
+### Where to start?
+
+Galaxy is a complicated project, with many moving parts. At first, it can be hard to figure out how it all works, or where your specific skill set and interests might fit. Essentially, there are two main activity areas - ***technical*** and ***community***.
+
+The distinction is only useful for organising things, because active two way interchange and overlap are always needed to keep the project on track. In practice, there are a number of contributors who work in both areas to ensure good communication and integration.
+
+### Technical Development
+
+A large scale, complicated, open source software engineering project and [source code](https://github.com/galaxyproject) forms the core of the project deliverables.  Like the project itself, the code repository is complicated and made up of many interacting components and dependencies. The large community that has become engaged with that code now helps sustain the project, in part by providing a pool of talented external contributors, large enough to place Galaxy among the top 2% of OpenHub projects for [developer activity](https://www.openhub.net/p/galaxybx/factoids#FactoidAgeVeryOld)
+
+If you would like to learn more about how you can contribute technical expertise, The [Working Groups](https://galaxyproject.org/community/wg/) page is the best place to start as there are a wide variety of specialised areas to choose from. Contact the specific Working Group convenor to learn more.
+
+### Community Development
+
+Like the source code, the community provides opportunities for contributions, such as collaborations to improve the sustainability, usefulness and growth of the community itself. In many cases, these projects will also involve technical aspects, but they rely primarily on other specialised scientific and organisational skills and experience.
+
+The [Outreach and Training WG](https://galaxyproject.org/community/wg/) addresses many issues that are community centered. This WG may be of interest to potential contributors to educational and training activities because it includes [Galaxy Training Network](https://training.galaxyproject.org/) collaborations
```

Comment:
```suggestion
The [Outreach and Training WG](https://galaxyproject.org/community/wg/) addresses many issues that are community centered. This WG may be of interest to potential contributors to educational and training activities because it includes [Galaxy Training Network](https://training.galaxyproject.org/) collaborations.
```